pad_api_data/pad_etl/processor/enemy_skillset.py

```diff
@@ -730,22 +441,22 @@ class ESJammerChangeRandom(ESOrbChange):
     def __init__(self, skill: EnemySkillRef):
         self.random_count = int(params(skill)[1])
         from_attr = 'Random {:d}'.format(self.random_count)
-        to_attr = ATTRIBUTE_MAP[6]
+        to_attr = Describe_EN.ATTRIBUTE_MAP[6]
```

Comment:
I was going to say that you should use the raw values here as opposed to the resolved strings, because this makes translations later difficult.

But the 'random x' makes things kind of tricky doesn't it? I think probably this needs to be its own object, like, ColorRef() with a ColorAndCountRef() subclass.

An annoying overhaul, I'm OK with leaving it out for now.